src/runner.c

```diff
@@ -109,13 +111,13 @@ void parse_args(struct config *cfg, int argc, char *argv[])
 			break;
 		/* scheduler times are ns, user gives ms */
 		case 'r':
-			cfg->attr.sched_runtime = 1000*1000*atoi(optarg);
+			cfg->attr.sched_runtime = MILLION * atoi(optarg);
```

Comment:
this variable should not be called "million", it should be called NANOSECONDS or something else. That what you want to do

Reply:
I will try to make up something smart

Reply:
I do not think there is something smarter than million (except maybe E6 etc.)

The thing is: We want the user to be able to hand over time parameters, as milliseconds, whereas the kernel interprets these integers as nanoseconds. Therefore, we have to multiply by one million. The only other option I see is calling it something like `MS_TO_NS_FACTOR` or something akin